docs: v1.14.1 changelog [skip ci] (#6222)
DevExpress/testcafe: CHANGELOG.md

# Changelog

## v1.14.1 (2021-05-12)

## Bug Fixes

> * Fixed a bug where TestCafe hangs during TestCafe Studio tests execution ([#5207](https://github.com/DevExpress/testcafe/issues/5207))

Reviewer: ```suggestion
* Fixed a bug that caused TestCafe to hang during the execution of TestCafe Studio tests ([#5207](https://github.com/DevExpress/testcafe/issues/5207))
```

* Added missing `disableScreenshots` type definitions ([#5735](https://github.com/DevExpress/testcafe/issues/5735))

> * Fixed an error where the unexpected popup browser window makes TestCafe hang ([#6132](https://github.com/DevExpress/testcafe/issues/6132))

Reviewer: ```suggestion
* Fixed a multi-window mode bug that caused TestCafe to launch an empty browser window and then hang ([#6132](https://github.com/DevExpress/testcafe/issues/6132))
```

> * Fixed switching to an `iframe` with the `srcdoc` attribute ([#6033](https://github.com/DevExpress/testcafe/issues/6033))

Reviewer: ```suggestion
* Fixed a bug that denied users access to iframes with `srcdoc` attributes ([#6033](https://github.com/DevExpress/testcafe/issues/6033))
```

> * Fixed a bug where Word online document is not fully loaded with Firefox ([testcafe-hammerhead#2287](https://github.com/DevExpress/testcafe-hammerhead/issues/2287))

Reviewer: ```suggestion
* Fixed a bug that interfered with the loading of Word Online documents in Firefox ([testcafe-hammerhead#2287](https://github.com/DevExpress/testcafe-hammerhead/issues/2287))
```
